### ISSUE TITLE
How many images did you use to train?

### ISSUE THREAD

**Issue by thimabru1010** [mohuazheliu/ResUnet-a#2]
I am trying to implement the multitasking version in keras on tf 2.0 and wanted some parameters to compare.

**Comment by ChienWong** [comment]
Probably totals 20000 images, and batch size is 4

**Comment by thimabru1010** [comment]
Ok, thank you! And did u try to implement the multitasking versions? I am trying to implement but tensors are having different sizes than labels...

**Comment by thimabru1010** [comment]
Why In Tanimoto loss, on denominator part, did you only square predictions and not labels? Because, following the Dice loss 3 in the paper, which is the Tanimoto loss, it suggests you to square predictions and labels and them sum each of these squared values, and not sum prediction values.

And why do you take the mean at the end of Tanimoto loss?

**Comment by ChienWong** [comment]
Sorry,I don't have plan to implement it in tf2.0 as this is a engineering project that the server don't use tf2.0.
About Tanimoto loss,maybe I make some mistakes in it that I caculate Tanimoto loss aim at every prediction of single pixel then sum them,so the square of label is themselves.
